### PR TITLE
fix(ADA-1084): Color contrast of v7 player icons

### PR DESCRIPTION
**Pull request by Tzipi-kaltura**
### Description of the Changes

**Issue:**
menu controls can have very low contrast ratios

**Fix:**
make the gradient stronger and add a shadow to the icons and the text

#### Resolves [ADA-1084](https://kaltura.atlassian.net/browse/ADA-1084)

[ADA-1084]: https://kaltura.atlassian.net/browse/ADA-1084?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/bottom-bar/_bottom-bar.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .player .bottom-bar {
-  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.6) 100%);
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0.00) 0%, rgba(0, 0, 0, 0.80) 100%);
```
Comment (SivanA-Kaltura) [inline src/components/bottom-bar/_bottom-bar.scss:2]: why not just rgba(0, 0, 0, 0) ?

File: src/components/top-bar/_top-bar.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .player .top-bar {
-  background: linear-gradient(0deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.6) 100%);
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0.80) 0%, rgba(0, 0, 0, 0.00) 100%);
```
Comment (SivanA-Kaltura) [inline src/components/top-bar/_top-bar.scss:2]: same, why not just rgba(0, 0, 0, 0)

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/bottom-bar/_bottom-bar.scss (hunk 1 above)
Comment (Tzipi-kaltura) [reply src/components/bottom-bar/_bottom-bar.scss:2]: it's for the gradient, it's started with dark background and in the end it's light.

File: src/components/top-bar/_top-bar.scss (hunk 2 above)
Comment (Tzipi-kaltura) [reply src/components/top-bar/_top-bar.scss:2]: same, just the opposite

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**